reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md

```diff
@@ -139,38 +137,39 @@ WS-Management service is running on the remote host and configured to listen
 for requests on the correct port and HTTP URL.
 ```
 
-To configure the listeners for all computers in a domain, enable the "Allow
-automatic configuration of listeners" policy in the following Group Policy
+To configure the listeners for all computers in a domain, enable the **Allow
+automatic configuration of listeners** policy in the following Group Policy
 path:
 
-Computer Configuration\\Administrative Templates\\Windows Components
-\\Windows Remote Management (WinRM)\\WinRM service
+```
+Computer Configuration\Administrative Templates\Windows Components
+\Windows Remote Management (WinRM)\WinRM service
```

Comment:
I think this is a single path, but it got broken into two lines.  Maybe recombine?

Reply:
Stet - broken due to About_ topic line limitations